docs/source/conf.py

```diff
@@ -97,6 +97,8 @@
 
 html_logo = '../image/optuna-logo.png'
 
+html4_writer = True
```

Comment:
### Without `html4_writer = True`:

<img width="702" alt="Screen Shot 2020-06-26 at 22 56 26" src="https://user-images.githubusercontent.com/17039389/85865111-5d1e5080-b800-11ea-8780-155daab9408f.png">

### With `html4_writer = True`:

<img width="702" alt="Screen Shot 2020-06-26 at 22 55 39" src="https://user-images.githubusercontent.com/17039389/85865170-71fae400-b800-11ea-8e28-d4c167021fe5.png">

One of the `sphinx_rtd_theme` core members says the second image is how it should look like.

https://github.com/readthedocs/sphinx_rtd_theme/issues/949#issuecomment-648942868

Reply:
Thank you for your investigation. As far as I understand, we do not need to introduce `html4_writer` as the following reasons:
- Unlike https://github.com/readthedocs/sphinx_rtd_theme/issues/949,  the descriptions of parameters do not exceed the width of the container.
- `html4_writer` will be deprecated as discussed in https://github.com/readthedocs/sphinx_rtd_theme/issues/938.

What do you think?

Reply:
That makes sense. I have reverted the change.